images/go/configs/latest-1.20.apko.yaml

```diff
@@ -0,0 +1,19 @@
+contents:
+  repositories:
+    - https://packages.wolfi.dev/os
+  keyring:
+    - https://packages.wolfi.dev/os/wolfi-signing.rsa.pub
+  packages:
+    - ca-certificates-bundle
+    - busybox
+    - build-base
+    - git
+    - openssh
+    - go-1.20
+
+environment:
+  GODEBUG: tarinsecurepath=0,zipinsecurepath=0
```

Comment:
Also I took the liberty of adding these environment variables, they enable some new security features in go 1.20!